fix:Fix/sensor fusion kit v2

tetsuya-t4 opened the pull request:
Fixed following 
- TIER IV camera driver v1.4.3 to v2.0.0
- fix kernel entry C1 to C2 on ansible setup script
- fix network interface name eth0 to enP1p1s0
- change ros network to localhost
- add multicast setting to localhost
- fix individual_params commit tag
- add download script about centerpoint

The following was confirmed
After installing [Anvil Jetpack 6.1 V002,](https://connecttech.com/ftp/Drivers/CTI-L4T-ORIN-AGX-36.4.0-V002.tgz) and run [the setup script procedure](https://tier4.github.io/edge-auto-docs/getting_started/sensor_fusion_kit_v2_getting_started_guide/02_installation.html) and confirm that roi cluster fusion and calibration tools are working.

tetsuya-t4 committed:
Add ros2 build dependencies

tetsuya-t4 committed:
fix driver link to v1.4.3 to v2.0.0

tetsuya-t4 committed:
remove post process for ~v1.5.0

tetsuya-t4 committed:
add DEFAULT kernel entry

tetsuya-t4 committed:
add :

tetsuya-t4 committed:
fix device tree setting in ansible

tetsuya-t4 committed:
fix ros network interface setting to localhost

tetsuya-t4 committed:
add setting `fsync_mfp`setting for use trigger mode on anvil

tetsuya-t4 committed:
change reference repository for l4t 36.4 and Anvil
- sensor-trigger
- ros2_v4l2_camera

tetsuya-t4 committed:
add image_geometry build command to `build.sh`
- for avoid runtime segmentation fault error cause of image_geometry package

tetsuya-t4 committed:
add centerpoint model data install scripts to ansible script

tetsuya-t4 committed:
fix autoware.repos
- update launcher repository(for integrate launch repo)
- update individual params repo
- add calibration depend repos
- add vision_opencv repo
  - for avoid runtime segmentation fault error cause of image_geometry package

tetsuya-t4 committed:
update network interface name `eth0` to `enP1p1s0`

tetsuya-t4 committed:
fix setup scripts
- fix individual_params commit tag
- fix ptp4l_host setting for lidar
- fix multicast setting for ros

pre-commit-ci[bot] committed:
style(pre-commit): autofix

manato commented on ansible/roles/artifacts/tasks/main.yaml:
```diff
@@ -76,3 +76,77 @@
     checksum: sha256:3d93ca05f31b63424d7d7246a01a2365953705a0ed3323ba5b6fddd744a4bfea
+
+# lidar_centerpoint
+- name: Create lidar_centerpoint directory inside {{ data_dir }}
```
@tetsuya-t4 
Since `data_dir` is not defined anywhere (this is directly supplied by the command line option in `setup-dev-env.sh`), would you mind defining this variable in `ansible/setup.yaml`? It looks `artifacts_script_download_dir` defined in the setup.yaml is good candidate to be replaced. When you replace the definition in the setup.yaml, it would be nice to get home directory using ansible function rather than direct specification like this:
```diff
-    - artifacts_script_download_dir: /home/autoware/autoware_data
+    - artifacts_script_download_dir: "{{ ansible_env.HOME }}/autoware_data"

manato replied:
@tetsuya-t4 
Thank you for your update. My previous comment intended to point out not only using `{{ ansible_env.HOME }}`, but also defining `data_dir` variable in `ansible/setup.yaml`. 

Because `data_dir` is used in `roles/artifacts/tasks/main.yaml` without variable definition, I'd appreciate it if you replace `data_dir` with `artivacts_script_download_dir` or vice versa.

manato commented on ansible/roles/ros2/tasks/main.yaml:
```diff
@@ -57,6 +57,17 @@
+  become: true
+  ansible.builtin.apt:
+    name:
+      - ros-humble-camera-info-manager
```
It would be nice to use a distribution-control variable rather than specifying directly.
```suggestion
      - ros-{{ rosdistro }}-camera-info-manager
```

manato commented on ansible/roles/ros2/tasks/main.yaml:
```diff
@@ -57,6 +57,17 @@
+    name:
+      - ros-humble-camera-info-manager
+      - ros-humble-camera-calibration-parsers
+      - ros-humble-cudnn-cmake-module
```
If `cudnn-cmake-module` is required to be installed manually here, I guess `tensorrt_cmake_module` is also required to be installed manually. I also think this can be installed via `rosdep install`  in the later phase (I checked the list of installed package via rosdep by ``rosdep keys --from-paths `colcon list --packages-up-to edge_auto_jetson_launch -p` ``

manato commented on ansible/roles/tier4_anvil_c2/defaults/main.yml:
It is perhaps possible to delete this `tier4_anvil_c2` role judging from the commit message of https://github.com/tier4/edge-auto-jetson/pull/70/commits/5049e41124516ea16d526a49d3d0f2d08df41050

manato reviewed (commented):
@tetsuya-t4 
Thank you for your thorough corrections. I noticed some points where improvement  could potentially be made, so I'd appreciate it if you could consider them

tetsuya-t4 committed:
fix ansible scripts
- remove `tier4_anvil_c2` roles
- remove unnecessary role(`Install ros2 dependencies` role)
- use `setup.yaml` variables for autoware_data download dir

* Verify that it works without any issues on Anvil + Jetpack6.1 V002

tetsuya-t4 commented:
@manato 
Thank you for your review. I have fixed the problem you pointed out and confirmed that it works, so please review again.

tetsuya-t4 committed:
Fix to use variables `artivacts_script_download_dir`

- chnage variable `data_dir` to `artivacts_script_download_dir`
- remove --extra-vars (`data_dir`)

tetsuya-t4 commented:
@manato 
Thank you for confirmation and I apologize for late response.I have corrected the part you pointed out, so please check it. Sorry for bothering you.

manato reviewed (approved):
@tetsuya-t4 
LGTM! Thank you for updating!

tetsuya-t4 committed:
fix autoware.repos

- fix `nebula`, `transport_drivers`, `ros2_socketcan`, `tensorrt_cmake_module` git version

tetsuya-t4 committed:
chore: update image_transport_plugins version

tetsuya-t4 committed:
build: improve build.sh

- Add jetson_clocks and parallel build flag (-j8)
- Add missing cmake linker flags (-lgmp, -lmpfr)
- Allow overriding cv_bridge instead of ignoring it

tetsuya-t4 committed:
feat: add python utility for inverse transform calculation

pre-commit-ci[bot] committed:
style(pre-commit): autofix

tetsuya-t4 committed:
add: create ros environment script and install ros workspace build dependencies script

pre-commit-ci[bot] committed:
style(pre-commit): autofix
